style.css

```diff
@@ -155,25 +164,30 @@ html {
 	justify-content: center;
 	font-family: 'Varela Round', sans-serif;
 	font-size: 3rem;
+	padding-top: 50px;
+	padding-bottom: 75px;
+	margin: 0;
 }
 
 .container {
```

Comment:
theres an issue with the height of this container, it's running into the next section when in mobile, I think you might like to adjust the height of your container in your media query, see image: 
![image](https://user-images.githubusercontent.com/45392468/149965452-d9fc501f-1598-4201-89a1-9abacb790f08.png)